src/fklearn/causal/cate_learning/double_machine_learning.py

```diff
@@ -151,7 +152,7 @@ def p(new_df: pd.DataFrame) -> pd.DataFrame:
         'prediction_column': prediction_column,
         'package': "sklearn",
         'package_version': sk_version,
-        'feature_importance': dict(zip(features, model_final_fitted.feature_importances_)),
+        'feature_importance': None,
```

Comment:
There isn't a standard in sklearn models for a feature importance attribute, which causes error depending on the model you use as the final model.